contracts/Overtime/SportsAMMV2.sol

```diff
@@ -337,13 +345,16 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
         amountsToBuy = new uint[](numOfMarkets);
         uint maxSupportedOdds = riskManager.maxSupportedOdds();
 
+        uint addedPayoutPercentage = addedPayoutPercentagePerCollateral[_collateral];
```

Comment:
This way the bonus percentage increases with games, even if they are small odd games. 
Another way to make it fixed per Ticket on the totalQuote, we can modify just this line:
`uint addedPayoutPercentage = addedPayoutPercentagePerCollateral[_collateral] / numOfMarkets;`

Reply:
its done like that intentionally to encourage THALES usage and multi leg parlays